Assets/Game/GUI/Scripts/MenuSystem/MenuNavigator.cs

```diff
@@ -28,35 +28,40 @@ protected MenuNode Current
     void Start()
     {
         Current = start;
-        InputMap.Instance.inputEnabled = true;
```

Comment:
Why did you remove this?

Reply:
i think it was an accident i will put it back